components/common/src/main/java/org/cardanofoundation/ledgersync/common/common/constant/Constant.java

```diff
@@ -28,6 +28,7 @@ public static boolean isLoveLace (byte[] data){
   public static boolean isTestnet(int networkMagic) {
     return networkMagic == TESTNET
```

Comment:
@Sotatek-HuyLe3a I think we can change this logic to  networkMagic != MAINNET, then it's Testnet

Reply:
I just updated